arb-defs.h

```diff
@@ -0,0 +1,148 @@
+/*
+    Copyright (C) 2022 Albin Ahlbäck
+    Copyright (C) 2022 Fredrik Johansson
```

Comment:
As I really haven't contributed to this file's content, feel free to remove me here. Also feel free to change the order (I simply did alphabetical order (gissa vem som alltid var högst upp på klasslistan :wink:)).